<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <widget class="QLCDNumber" name="stage_temp_display">
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heater Target (C)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="tank_temp_display">
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="tank_label">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>tank temperature (C)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="heater_target">
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="stage_label">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>stage temperature (C)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heater P</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QwtThermo" name="heater_P" native="true">
        <property name="minValue" stdset="0">
         <double>-1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QwtThermo" name="heater_D" native="true">
        <property name="minValue" stdset="0">
         <double>-1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heater I</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QwtThermo" name="heater_output" native="true">
        <property name="maxValue" stdset="0">
         <double>100.000000000000000</double>
        </property>
        <property name="minValue" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heater Output (%)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Heater D</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QwtThermo" name="heater_I" native="true">
        <property name="minValue" stdset="0">
         <double>-1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="wavelength_label">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wavelength (nm)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLCDNumber" name="wavelength_display">
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="wavelength_label_2">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CCD Power</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLCDNumber" name="VBB_display">
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="wavelength_label_3">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Back Bias voltage (V)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="wavelength_label_4">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Experiment Status</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>auto update timeout (ms)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="updatetimeout">
      <property name="minimum">
       <number>500</number>
      </property>
      <property name="maximum">
       <number>20000</number>
      </property>
      <property name="value">
       <number>2000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionManual_update"/>
    <addaction name="actionAuto_update"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionManual_update">
   <property name="text">
    <string>&amp;manual update</string>
   </property>
  </action>
  <action name="actionAuto_update">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;auto update</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
